File: Leetcode/Long-Pressed_Name.java
class Solution {
    public boolean isLongPressedName(String name, String typed) {
        int i = 0;
        for(int j = 0; j < typed.length(); j++){
            if(i < name.length() && name.charAt(i) == typed.charAt(j)){
                i++;
            }
        }
        
        return i == name.length();
    }
}

File: Leetcode/Long-pressed_Name.py
class Solution(object):
    def isLongPressedName(self, name, typed):
        """
        :type name: str
        :type typed: str
        :rtype: bool
        """
        i = 0
        
        for j in range(len(typed)):
            if (i < len(name) and name[i] == typed[j]):
                i += 1
        
        return i == len(name)
